apps/recovery-portal/src/main/webapp/self-registration-without-verification.jsp

```diff
@@ -316,23 +327,11 @@
                     return false;
                 }
 
-                <%
-                if(reCaptchaEnabled) {
-                %>
-                var resp = $("[name='g-recaptcha-response']")[0].value;
-                if (resp.trim() == '') {
-                    error_msg.text("<%=IdentityManagementEndpointUtil.i18n(recoveryResourceBundle,
-                    "Please.select.reCaptcha")%>");
-                    error_msg.show();
-                    $("html, body").animate({scrollTop: error_msg.offset().top}, 'slow');
-                    return false;
-                }
-                <%
-                }
-                %>
-
                 return true;
+
             });
+
+
```

Comment:
Let's remove this extra line.

Reply:
Fixed in https://github.com/wso2/identity-apps/pull/3298